etc/terraform-arm64.conf

```diff
@@ -0,0 +1,33 @@
+# target architecture - i386, amd64 or all
```

Comment:
I don't think we release stable arm64 images soon because it needs testing, could you rename this file to `etc/terraform-daily-8.0-arm64.conf`?

Reply:
Also you should update the comment `i386, amd64 or all` in terraform*.conf.

Reply:
I believe `etc/terraform-arm64.conf` (similar to `etc/terraform.conf`) is intended for local builds and testing. Perhaps I should add a `etc/terraform-daily-8.0-arm64.conf` to align with `etc/terraform-daily-8.0-azure.conf`?

Reply:
Thanks for adding `etc/terraform-daily-8.0-azure-arm64.conf`.

I think we should add `arm64` to the `# target architecture - i386, amd64 or all` comment at least in the OS 8 confs (`terraform-arm64.conf`, `terraform-daily-8.0-azure-arm64.conf`, `terraform-daily-8.0-azure.conf`, `terraform-stable-8.0-azure.conf`, and `terraform.conf`). However, we should review this parameter anyway because we should no longer support `i386` so we can revisit this later in another PR.